core/block/delete.go

```diff
@@ -78,6 +78,7 @@ func (s *Service) DeleteObjectByFullID(id domain.FullID) (err error) {
 	}
 
 	sendOnRemoveEvent(s.eventSender, id.ObjectID)
+	sendOnCloseEvent(s.eventSender, id.ObjectID)
```

Comment:
can we test it?

Reply:
Yeah, I have added unit-test on event broadcasting and also moved deletion logics of derived objects to separate func